cpp/src/io/csv/csv_reader.cu

```diff
@@ -750,7 +750,20 @@ gdf_error read_csv(csv_read_arg *args)
 		for ( int x = 0; x < raw_csv->num_actual_cols; x++) {
 
 			std::string temp_type 	= args->dtype[x];
-			gdf_dtype col_dtype		= convertStringToDtype( temp_type );
+			gdf_dtype col_dtype;
+			if(temp_type.find(':') != std::string::npos){
+			    for (auto it = raw_csv->col_names.begin(); it != raw_csv->col_names.end(); it++){
+			        std::size_t idx = temp_type.find(':');
+                    if(temp_type.substr( 0, idx) == *it){
```

Comment:
Please correct the indentation.  Use whatever whitespace is being used in the rest of the file (tabs or spaces), please don't mix.

Reply:
I have fixed it. thanks